core/src/main/java/com/linecorp/armeria/server/RouteBuilder.java

```diff
@@ -413,6 +417,14 @@ RouteBuilder matchesHeaders(List<RoutingPredicate<HttpHeaders>> headerPredicates
         return this;
     }
 
+    /**
+     * Sets the {@link Route#isFallback()} flag.
+     */
+    RouteBuilder fallback() {
```

Comment:
Can this be private? Or maybe it's too much to have a flag instead of just passing in false in the builder and invoking the constructor with true for the constant.

Reply:
Indeed. Let me make `isFallback` final and initialize it in the constructor.